src/tools/rest-backend/.gitignore

```diff
@@ -0,0 +1,3 @@
+CMakeCache.txt
+nbproject/
```

Comment:
nbproject is for project files from netbeans. If we do not have it elsewhere in our gitignores, we should keep it.

No, I did not use in-source builds for the backend. Why?